docs/reports/create-a-draft.md

```diff
@@ -0,0 +1,59 @@
+
+## Create a draft report
+After logging into the [administration area](https://reports.opensafely.org/admin/) click "Add" next to Reports:
+
+![Add report button](./images/reports-admin-add-report.jpg)
+
+### Organisation
+Select your organisation from the list.
+If it's missing from the list, please [contact us](../how-to-get-help.md).
+
+
+### Navigation
+Pick the `Category` you want to host your report under in the side nav of the site.
+
+Then set a `Menu name`.
+
+
+### Report file details (GitHub)
+!!! note
+    It was previously possible to release outputs to GitHub and the reports site was originally built to use those outputs.
+    OpenSAFELY has moved away from this method and all outputs are now released to the jobs site.
+    Please make sure to use the jobs site field in this form for your output location instead of the GitHub ones.
+
+
+### Report file details (Jobs site)
+Find your workspace on [the jobs site](https://jobs.opensafely.org).
+Click `Released Outputs` to view the most recent version of each of your released outputs:
+
+![Most recent version of released outputs](./images/job-server-workspace-latest-outputs.jpg)
+
+Select the file you want to make your report with from the list on the left, and copy the direct URL which shows at the top of the file viewer.  This is the URL you will need in the reports admin.
+
+![Where to get the direct URL for a file](./images/job-server-direct-output-file-link-release.jpg)
+
+
+### Front matter
+These fields are displayed above your report on the site.
+
+
+### DOI
```

Comment:
What DOI does this relate to? The report itself, the paper etc?

Reply:
Jess has suggested we see how this works with the pilot users so we'll revist it once they've tried it out!